api/test_api.py

```diff
@@ -134,6 +133,11 @@ def test_get_filter_queries_alert_is_first_filter():
         "WHERE a.alert_value = 'alert'",
         "mt.magtype_value = 'magtype'",
         "t.type_value = 'type'",
-        "e.magnitude >= 'min mag'",
-        "NOT YET IMPLEMENTED"
+        "e.magnitude >= 'min mag'"
     ]
+
+
+def test_filter_by_continent(example_api_response):
+    assert filter_by_continent(example_api_response, "Africa") == []
+    assert filter_by_continent(
+        example_api_response, "North America") == example_api_response
```

Comment:
you can try using pytest.parametarize to test several inputs and outputs in a single function

Reply:
True, I will consider this when I finalise testing.